angular-portfolio/src/app/awards/awards.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { AwardsService } from '../awards.service';
+import { Awards } from '../awardsTemplate';
+
+@Component({
+  selector: 'app-awards',
+  templateUrl: './awards.component.html',
+  styleUrls: ['./awards.component.scss']
+})
+export class AwardsComponent implements OnInit {
+
+  awards: Awards[];
+  selectedAward: Awards;
+
+  constructor(private awardService: AwardsService) { }
+
+  ngOnInit(): void {
+    this.getAwards();
+  }
+
+  getAwards(): void {
+    this.awardService.getAwards()
+      .subscribe(awards => this.awards = awards);
```

Comment:
Use a "block body" here instead of an "expression body". See https://g3doc.corp.google.com/javascript/typescript/g3doc/styleguide.md?cl=head#function-expressions for details.

Reply:
I don't have access to that resource. I tried switching form arrow syntax to normal function but did not seem to work.

Reply:
Disregard last comment. You explained it further on.